### ISSUE TITLE
Don't support python file with Chinese characters.

### ISSUE THREAD

**Issue by lijiawei-1** [cmcpasserby/MayaCharm#68]
Using pycharm 2021.3.3, Maya 2023.
If python file have chinese character, when using Run config to run file in maya, will get a error:

```
# Error: line 1: 'gbk' codec can't decode byte 0xad in position 8957: illegal multibyte sequence
# # Traceback (most recent call last):
# #   File "<maya console>", line 1, in <module>
# # UnicodeDecodeError: 'gbk' codec can't decode byte 0xad in position 8957: illegal multibyte sequence
```

python file starts with `# -*- coding: UTF-8 -*-`

**Comment by lijiawei-1** [comment]
If I change first line to `# -*- coding: gbk -*-`, there won't be that error.
So mayaCharm doesn't recognize my file coding?

**Comment by cmcpasserby** [comment]
will fix this as its a duplicate of #69 over the weekend.

**Comment by 44hero** [comment]
Thank you very much for your response to our request for correction.
